app/src/test/java/com/fstyle/structure_android/data/source/UserRepositoryTest.java

```diff
@@ -29,78 +28,62 @@
     private static final String USER_LOGIN_1 = "abc";
     private static final String USER_LOGIN_2 = "def";
 
-    @Mock
-    NameApi mNameApi;
-
+    @InjectMocks
     private UserRepository mUserRepository;
+    @Mock
+    UserLocalDataSource mLocalDataSource;
+    @Mock
+    UserRemoteDataSource mRemoteDataSource;
 
     @Before
     public void setUp() throws Exception {
         MockitoAnnotations.initMocks(this);
-        mUserRepository = new UserRepository(null, new UserRemoteDataSource(mNameApi));
-    }
-
-    @After
-    public void tearDown() throws Exception {
-
     }
 
     @Test
-    public void searchUsers200ResponseCodeInvokesCorrectApiCalls() {
+    public void searchUsers_200ResponseCode_InvokesCorrectApiCalls() {
```

Comment:
ERROR: [MethodNamingConventions] Method names should not contain underscores
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#MethodNamingConventions